.travis.yml

```diff
@@ -38,6 +38,8 @@ matrix:
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration RUNC_USE_SYSTEMD=yes'
         # same setup but with fs2 driver instead of systemd
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration'
+        # rootless
+        - sudo ssh default -t 'cd /vagrant && sudo make localrootlessintegration'
```

Comment:
Do we want to run the same tests with `RUNC_USE_SYSTEMD=yes`?

Reply:
No until https://github.com/opencontainers/runc/pull/2281 gets merged.
And https://github.com/opencontainers/runc/pull/2281 will require this to be merged.

Reply:
Right, not in this PR